site/src/components/SdkPage.js

```diff
@@ -31,6 +31,13 @@ export default function SdkPage({Svg, title, repoLinks, samplesLink}) {
                                 <td>Code Samples showing how to utilize the Bandwidth {title} SDK</td>
                                 <td><GitHubLink GHLink={samplesLink}/></td>
                             </tr>
+                            { migrationGuides == true &&
+                                <tr className={sdkStyles.sdkMigrationGuides}>
+                                    <td />
+                                    <td><a href={"/migration-guides/" + title.toLowerCase()}>Migration Guides</a></td>
```

Comment:
```suggestion
                                    <td><a href={"/migration-guides/" + title.toLowerCase()}>Migration Guide</a></td>
```
I think here this should just say guide because it's linking to the individual guide for the selected language